### DISCUSSION TITLE
KEDA HTTP Add-On Chart v0.2.2

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#235] [Announcements]
*Chart: v0.2.2 | App: 0.2.0*

Add-on for KEDA to scale HTTP workloads.

## TL;DR

```shell
helm repo add keda https://kedacore.github.io/charts
helm install keda/keda
helm install http-add-on keda/keda-add-ons-http
```

### Features

- Provide capability to use `nodeSelector`, `tolerations` and `affinity` (#212 - Contributed by @morganchristiansson 🎉)

### Fixes / Changes

- Add `targetPendingRequests` to CRD (#231 - Contributed by @avinash31d 🎉)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @whitesource-bolt-for-github made their first contribution in https://github.com/kedacore/charts/pull/234
* @avinash31d made their first contribution in https://github.com/kedacore/charts/pull/231

**Full Changelog**: https://github.com/kedacore/charts/compare/keda-add-ons-http-v0.2.1...keda-add-ons-http-v0.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/keda-add-ons-http-v0.2.2'>KEDA HTTP Add-On Chart v0.2.2</a>.</em>
